site/layouts/servermessage.hbs

```diff
@@ -1,3 +1,8 @@
+---
+{
+	"officiallanguage": "<%= language === 'en' || language === 'fr' %>"
+}
+---
```

Comment:
I've noticed that the line break just after this spot will be retained in generated HTML code. So in generated server message page templates, the file's first line will be empty, followed by the HTML doctype on the second line. It'll still pass HTML validation, but looks kind of odd.

Btw this issue seems to affect all handlebars files that contain [YAML front matter (YFM)](http://assemble.io/docs/YAML-front-matter.html). The line break after the second instance of ``---`` is always included in generated HTML code. It's only really obvious in this case since it precedes the page's doctype.

@nschonni @LaurentGoderre Thoughts?

Reply:
@nschonni @LaurentGoderre Any thoughts regarding my previous line comment?

Reply:
A leading line can also blow up old IE i believe. Or at least it automagically triggers compat mode

Reply:
@nschonni According to https://wet-boew.github.io/v4.0-ci/docs/versions/rdmp-en.html, WET 4.0 technically no longer supports IE9 and below. So that might not be as bad in practice as it could've been. Although I'd still prefer avoiding the line break.

All I can think of to easily avoid the line break would be to move ``<!DOCTYPE html>`` to the end of line 5 (which would place it right up against ``---``). Only drawback to it is that it'd look a bit strange to anyone that looks through this HBS file.

Thoughts?

Reply:
I'm gonna go ahead and place ``---`` and ``<!DOCTYPE html>`` side-by-side on the same line to omit the leading line break. Can't think of a more elegant way of getting rid of it atm.